easybuild/easyconfigs/m/MCU/MCU-3366c85-gomkl-2021a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'MCU'
+version = '3366c85'  # There is a 0.2 release but it's too old and it doesn't work
+
+homepage = 'https://hungqpham.com/mcu/'
+description = """A package for periodic wavefunction and crystallography analysis.
+ mcu is designed to support large scale analysis and topological descriptions for periodic wavefunction."""
+
+toolchain = {'name': 'gomkl', 'version': '2021a'}
+
+source_urls = ['https://github.com/hungpham2017/mcu/archive/']
+sources = [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}]
+checksums = ['96553924c57ca27d2e4d8668be2c97fc6019c139f91950a798733c5575a8b44a']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('matplotlib', '3.4.2'),
+    ('spglib-python', '1.16.1'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+options = {'modulename': 'mcu'}
```

Comment:
This line should not be required, so please remove it.

The `PythonPackage` easyblock will do an import test with the lowercase software name by default (see output of `eb --trace` when installing the easyconfig).